GSA-TTS/identity-site
# Upgrade to LGDS 9.0 to reset prose styling

## 🛠 Summary of changes

Updates to `@18f/identity-design-system@9.0.0-beta.2` (currently prerelease), which includes changes to reset prose styling. The initial implementation of this site included custom variant of the prose styles, presumably to work around some of the undesirable implications of the original styles.

As a draft, the intent is to assess potential impact and whether additional changes are necessary to the design system revisions before a final 9.0.0 release (as described at https://github.com/18F/identity-design-system/pull/390#issuecomment-1883779930).

This also reconfigures line-height to use a USWDS configurable value `1.75` value rather than an ad-hoc `1.875rem`, which may result in a slight reduction in line height.

## 📜 Testing Plan

- Go to https://federalist-17bd62cc-77b7-4687-9c62-39b462ce6fd5.sites.pages.cloud.gov/preview/18f/identity-site/aduth-rm-prose-styles/
- Check that "prose" content appears as expected:
   - ["What is Login.gov" page content](https://federalist-17bd62cc-77b7-4687-9c62-39b462ce6fd5.sites.pages.cloud.gov/preview/18f/identity-site/aduth-rm-prose-styles/what-is-login/)
   - Any help article's content ([example](https://federalist-17bd62cc-77b7-4687-9c62-39b462ce6fd5.sites.pages.cloud.gov/preview/18f/identity-site/aduth-rm-prose-styles/help/trouble-signing-in/how-to-sign-in/))
   - Any "sidenav" layout page content:
      - Any "Policy" section page content ([example](https://federalist-17bd62cc-77b7-4687-9c62-39b462ce6fd5.sites.pages.cloud.gov/preview/18f/identity-site/aduth-rm-prose-styles/policy/))
      - ["Contact Us" page](https://federalist-17bd62cc-77b7-4687-9c62-39b462ce6fd5.sites.pages.cloud.gov/preview/18f/identity-site/aduth-rm-prose-styles/contact/)
      - ["Join us" page](https://federalist-17bd62cc-77b7-4687-9c62-39b462ce6fd5.sites.pages.cloud.gov/preview/18f/identity-site/aduth-rm-prose-styles/join/)
      - ["About us" page](https://federalist-17bd62cc-77b7-4687-9c62-39b462ce6fd5.sites.pages.cloud.gov/preview/18f/identity-site/aduth-rm-prose-styles/about-us/)
      - ["Accessibility" page](https://federalist-17bd62cc-77b7-4687-9c62-39b462ce6fd5.sites.pages.cloud.gov/preview/18f/identity-site/aduth-rm-prose-styles/accessibility/)
